versioning/BRM/extentmap.cpp

```diff
@@ -2339,24 +2303,20 @@ int ExtentMap::lookupLocal_DBroot(int OID, uint16_t dbroot, uint32_t partitionNu
     grabEMEntryTable(READ);
     grabEMIndex(READ);
 
-    entries = fEMShminfo->allocdSize / sizeof(struct EMEntry);
-
-    for (i = 0; i < entries; i++)
+    for (auto emIt = fExtentMapRBTree->begin(), end = fExtentMapRBTree->end(); emIt != end; ++emIt)
```

Comment:
This is my bad that I didn't fix this routine.
Could I ask you for a favor to apply index search here?